packages/skills/carpark_client/handlers.py

```diff
@@ -247,11 +246,10 @@ def handle(self, message: Message) -> None:
         """
         # convenience representations
         oef_msg = cast(OEFMessage, message)
-        oef_msg_type = OEFMessage.Type(oef_msg.get("type"))
+        oef_msg_type = oef_msg.type
```

Comment:
not sure why we assign to this variable here, seems unnecessary, just use below directly